lms/djangoapps/courseware/views/index.py

```diff
@@ -438,13 +438,6 @@ def _create_courseware_context(self, request):
         # entrance exam data
         self._add_entrance_exam_to_context(courseware_context)
 
-        # staff masquerading data
-        if not check_course_open_for_learner(self.effective_user, self.course):
```

Comment:
After this change, is `check_course_open_for_learner()` used anywhere?

Reply:
this file calls check_course_access which calls has_access which calls check_course_open_for_learner.  
but i don't think this file calls check_course_open_for_learner directly anymore, so I'm surprised it did not raise a pylint error  
I guess I can remove the import

Reply:
I was actually wondering about removing the function entirely... it sounds like it is still used elsewhere though.